<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_curtime">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="mainlayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="minimumSize">
         <size>
          <width>610</width>
          <height>0</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>系统信息</string>
         </attribute>
         <widget class="QWidget" name="gridLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>-1</x>
            <y>5</y>
            <width>601</width>
            <height>217</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="7" column="1">
            <widget class="QLabel" name="label_osver">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>        CPU名称：</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>        CPU类型：</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_8">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>        CPU频率：</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLabel" name="label_ostype">
             <property name="text">
              <string>Linux</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_10">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>操作系统信息</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_11">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>        操作系统类型：</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_13">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>        操作系统版本：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>处理器信息</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QLabel" name="label_gcc">
             <property name="text">
              <string>unknow</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_cpuname">
             <property name="text">
              <string>unknow</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="label_cputype">
             <property name="text">
              <string>unknow</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="label_cache">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_35">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>        Cache大小：</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="label_cpuhz">
             <property name="text">
              <string>unknow</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>        GCC编译器版本：</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QCustomPlot" name="customPlot" native="true">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>230</y>
            <width>531</width>
            <height>111</height>
           </rect>
          </property>
          <widget class="QLabel" name="label_32">
           <property name="geometry">
            <rect>
             <x>240</x>
             <y>0</y>
             <width>67</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>CPU</string>
           </property>
          </widget>
          <zorder>label_32</zorder>
          <zorder>label_32</zorder>
          <zorder>label_32</zorder>
          <zorder>label_34</zorder>
          <zorder>label_34</zorder>
         </widget>
         <widget class="QLabel" name="label_34">
          <property name="geometry">
           <rect>
            <x>550</x>
            <y>320</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_31">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>230</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>%</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_33">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>320</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>120s</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>内存资源</string>
         </attribute>
         <widget class="QCustomPlot" name="customplot_mem" native="true">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>30</y>
            <width>531</width>
            <height>101</height>
           </rect>
          </property>
          <zorder>label_usedmem</zorder>
          <zorder>label_totalmem</zorder>
          <zorder>label_2</zorder>
          <zorder>label_5</zorder>
          <zorder>label_freemem</zorder>
          <zorder>label_12</zorder>
         </widget>
         <widget class="QCustomPlot" name="customplot_swap" native="true">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>200</y>
            <width>531</width>
            <height>101</height>
           </rect>
          </property>
          <zorder>label_5</zorder>
          <zorder>label_12</zorder>
          <zorder>label_freemem</zorder>
          <zorder>label_usedmem</zorder>
          <zorder>label_usedmem</zorder>
         </widget>
         <widget class="QLabel" name="label_19">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>130</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>120s</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_24">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>300</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>120s</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_25">
          <property name="geometry">
           <rect>
            <x>550</x>
            <y>300</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_26">
          <property name="geometry">
           <rect>
            <x>550</x>
            <y>130</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_27">
          <property name="geometry">
           <rect>
            <x>290</x>
            <y>0</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>内存</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_28">
          <property name="geometry">
           <rect>
            <x>290</x>
            <y>170</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>交换分区</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_29">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>180</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>%</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_30">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>10</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>%</string>
          </property>
         </widget>
         <widget class="QWidget" name="gridLayoutWidget_3">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>140</y>
            <width>457</width>
            <height>25</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="2">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>      剩余：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>     已用：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="6">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>      共：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QLabel" name="label_freemem">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="0" column="7">
            <widget class="QLabel" name="label_usedmem">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_totalmem">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="gridLayoutWidget_4">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>310</y>
            <width>463</width>
            <height>31</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="10">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>     剩余：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="14">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>      共：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="15">
            <widget class="QLabel" name="label_totalswap">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>       已用：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="12">
            <widget class="QLabel" name="label_freeswap">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="0" column="8">
            <widget class="QLabel" name="label_usedswap">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <zorder>customplot_mem</zorder>
         <zorder>customplot_swap</zorder>
         <zorder>label_19</zorder>
         <zorder>label_24</zorder>
         <zorder>label_25</zorder>
         <zorder>label_26</zorder>
         <zorder>label_27</zorder>
         <zorder>label_28</zorder>
         <zorder>label_29</zorder>
         <zorder>label_30</zorder>
         <zorder>gridLayoutWidget_3</zorder>
         <zorder>label_totalmem</zorder>
         <zorder>gridLayoutWidget_4</zorder>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>进程信息</string>
         </attribute>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>591</width>
            <height>341</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QTableWidget" name="tableWidget">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>100</horstretch>
               <verstretch>100</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="midLineWidth">
              <number>0</number>
             </property>
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAsNeeded</enum>
             </property>
             <property name="autoScrollMargin">
              <number>20</number>
             </property>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>90</number>
             </attribute>
             <attribute name="verticalHeaderDefaultSectionSize">
              <number>30</number>
             </attribute>
            </widget>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="1">
              <widget class="QLabel" name="label_totalpro">
               <property name="text">
                <string>textlabel</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>运行进程：</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLabel" name="label_sleepro">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QLabel" name="label_zombie">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="label_runpro">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item row="2" column="3">
              <layout class="QGridLayout" name="gridLayout_5">
               <item row="0" column="1">
                <widget class="QPushButton" name="pushButton_2">
                 <property name="text">
                  <string>重启</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="2" column="1">
              <widget class="QPushButton" name="pushButton_3">
               <property name="text">
                <string>结束进程</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QPushButton" name="pushButton_4">
               <property name="text">
                <string>查找进程</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_20">
               <property name="text">
                <string>睡眠进程： </string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>总进程数：</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_22">
               <property name="text">
                <string>僵死进程：</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>关机</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QPushButton" name="pushButton_5">
          <property name="geometry">
           <rect>
            <x>430</x>
            <y>355</y>
            <width>134</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Search process</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="pro_text">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>365</y>
            <width>411</width>
            <height>61</height>
           </rect>
          </property>
          <property name="placeholderText">
           <string>input part of pid or name</string>
          </property>
         </widget>
         <zorder>layoutWidget</zorder>
         <zorder>pushButton_5</zorder>
         <zorder>pro_text</zorder>
         <zorder>label_22</zorder>
         <zorder>tableWidget</zorder>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>CPU使用率:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>内存使用率:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar_2">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>632</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
